lib/kitchen/directions/bake_preface/main.rb

```diff
@@ -3,8 +3,8 @@
 module Kitchen
   module Directions
     module BakePreface
-      def self.v1(book:)
-        V1.new.bake(book: book)
+      def self.v1(book:, title_element:)
```

Comment:
add an option because am gov is a div instead of h1

Reply:
is there a reason to define them seperately rather than giving the second argument a default?